current/twl/RP1-mapbox/RP1-mapbox.json

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
Why dont we have RP1, 100 and 1000 as variables for this dataset? Such that we can select them as dropdown in a viewer? Then the mapbox files with different RP would be three in the folder twl-mapbox (which is the variable). I think this has to do with the CF compliant file?

Reply:
Nicely fixed